docs/src/examples/components/Dropdown/Types/DropdownExampleSearchMultiple.shorthand.steps.ts

```diff
@@ -1,23 +1,23 @@
-import { Dropdown, List, Indicator, Input, Label } from '@stardust-ui/react'
+import { Dropdown, DropdownSearchInput } from '@stardust-ui/react'
+
+const selectors = {
+  toggleIndicator: `.${Dropdown.slotClassNames.toggleIndicator}`,
+  input: `.${DropdownSearchInput.slotClassNames.input}`,
+  item: (itemIndex: number) => `.${Dropdown.slotClassNames.itemsList} li:nth-child(${itemIndex})`,
+  selectedItem: (itemIndex: number) =>
+    `.${Dropdown.slotClassNames.selectedItems} span:nth-child(${itemIndex})`,
+}
 
 const steps: ScreenerSteps = [
   (steps, keys) =>
     steps
-      .click(`.${Dropdown.className} .${Indicator.className}`)
-      .click(`.${List.className} li:nth-child(2)`)
-      .click(`.${Dropdown.className} .${Indicator.className}`)
-      .click(`.${List.className} li:nth-child(2)`)
-      .keys(`.${Input.slotClassNames.input}`, keys.leftArrow)
+      .click(selectors.toggleIndicator)
+      .click(selectors.item(2))
+      .click(selectors.toggleIndicator)
+      .click(selectors.item(2))
+      .keys(selectors.input, keys.leftArrow)
       .snapshot('Selects last selected element'),
-  (steps, keys) =>
-    steps
-      .click(`.${Dropdown.className} .${Indicator.className}`)
```

Comment:
pretty sure that was a strange merge previously so I removed it, it was just repeating the steps above

Reply:
fixed